enterprise/models.py

```diff
@@ -41,6 +42,13 @@
 mark_safe_lazy = lazy(mark_safe, six.text_type)  # pylint: disable=invalid-name
 
 
+def mode_slugs():
+    """
+    Return course modes slugs.
+    """
+    return json.dumps(['verified', 'professional', 'no-id-professional', 'credit', 'audit', 'honor'])
```

Comment:
Possible to pick this up from https://github.com/edx/edx-enterprise/blob/master/enterprise/constants.py#L30? Maybe renaming the variable or something. Just so we don't have these things all over the place. You might also have to add `'credit'` to that one.